data/tinkerpop-no-edge-labels.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" ?><graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.1/graphml.xsd"><key id="labelV" for="node" attr.name="labelV" attr.type="string"></key><key id="name" for="node" attr.name="name" attr.type="string"></key><key id="lang" for="node" attr.name="lang" attr.type="string"></key><key id="age" for="node" attr.name="age" attr.type="int"></key><key id="weight" for="edge" attr.name="weight" attr.type="float"></key><graph id="G" edgedefault="directed"><node id="1"><data key="labelV">vertex</data><data key="name">marko</data><data key="age">29</data></node><node id="2"><data key="labelV">vertex</data><data key="name">vadas</data><data key="age">27</data></node><node id="3"><data key="labelV">vertex</data><data key="name">lop</data><data key="lang">java</data></node><node id="4"><data key="labelV">vertex</data><data key="name">josh</data><data key="age">32</data></node><node id="5"><data key="labelV">vertex</data><data key="name">ripple</data><data key="lang">java</data></node><node id="6"><data key="labelV">vertex</data><data key="name">peter</data><data key="age">35</data></node><edge id="7" source="1" target="2"><data key="weight">0.5</data></edge><edge id="8" source="1" target="4"><data key="weight">1.0</data></edge><edge id="9" source="1" target="3"><data key="weight">0.4</data></edge><edge id="10" source="4" target="5"><data key="weight">1.0</data></edge><edge id="11" source="4" target="3"><data key="weight">0.4</data></edge><edge id="12" source="6" target="3"><data key="weight">0.2</data></edge></graph></graphml>
```

Comment:
I think you can remove this file. We don't need to include that in our distributions. We really only need it for unit testing purposes.